docs/INBM Windows.md

```diff
@@ -1,6 +1,58 @@
+# Installing and Provisioning INBM for Windows in UCC mode
+
+This guide will walk you through the installation and provisioning of INBM for Windows in UCC mode, which installs the MQTT broker infrastructure and the Cloudadapter agent.
+
+## Prerequisites
+
+Before you begin, ensure that the following files contain the client and server IDs for UCC:
+
+- `C:\ucc\client_id`
+- `C:\ucc\server_id`
+
+Also ensure you are installing on Windows 10, our supported Windows version.
+
+## Installation
+
+1. Download or build the INBM for Windows zip file and extract its contents to `C:\`. This will create a directory called `C:\inb-files`. To build, see instructions for building INBM. The build output in `dist` will contain a file `inbm-windows.zip`.
+
+2. Open a Command Prompt with Administrator privileges. To do this, press `Win + X` and select "Command Prompt (Admin)" from the menu.
+
+3. Navigate to the `C:\inb-files\intel-manageability` directory by running the following command:
+
+   ```
+   cd C:\inb-files\intel-manageability
+   ```
+
+4. Set the environment variable `UCC_MODE` to `true` by running the following command:
+
+   ```
+   set UCC_MODE=true
+   ```
+
+5. Run the `install.ps1` script by executing the following command:
+
+   ```
+   powershell -ExecutionPolicy Bypass -File install.ps1
+   ```
+
+6. If you want to provide your own `adapter.cfg` instead of providing UCC provisioning details interactively, set the environment variable `NO_CLOUD` to `true` before running the `provision.ps1` script:
+
+   ```
+   set NO_CLOUD=true
+   ```
+
```

Comment:
Add a note here to point to the "In-Band Manageability User Guide - UCC.md" file to check out the interactive options.
I am assuming that these remain the same between Windows and Ubuntu. (if not, we should bring them to parity)